packages/workbox-webpack-plugin/src/lib/generate-webpack-sw.js

```diff
@@ -0,0 +1,29 @@
+const {readFile} = require('./utils/read-file');
```

Comment:
Is this the webpack templating that you've used instead of workbox-builds templating?

If so, renaming this to `webpack-generate-sw-template` would help with context.

Reply:
In favor of the naming Matt suggested above.